hermit-abi/src/lib.rs

```diff
@@ -101,6 +108,7 @@ pub const LOW_PRIO: Priority = Priority::from(1);
 pub struct Handle(usize);
 
 pub const NSEC_PER_SEC: u64 = 1_000_000_000;
+pub const FUTEX_RELATIVE_TIMEOUT: usize = 1;
```

Comment:
Should this not match the type of `flags`?

```suggestion
pub const FUTEX_RELATIVE_TIMEOUT: u32 = 1;
```

Reply:
Oh yes, of course. Thanks!